src/graph/entities/qg_edge.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#pragma once
+
+#include "edge.h"
+#include "qg_node.h"
+#include <string.h>
+
+struct QGEdge {
+    const char *alias;
+    const char **reltypes;
+    int *reltypeIDs; // TODO can be uint save for GRAPH_NO_RELATION
+    QGNode *src;
+    QGNode *dest;
+    uint minHops;   /* Minimum number of hops this edge represents. */
+    uint maxHops;   /* Maximum number of hops this edge represents. */
+    uint id;
+};
+
+typedef struct QGEdge QGEdge;
```

Comment:
Can we combine this typedef with the previous struct QGEdge, like we do for most of our structs.

Reply:
We produce a lot of Clang warnings when using different styles (since QGNodes and QGEdges refer to each other so much). This is the same logic as we have in `edge.h`, and I think we have to stick with it for now.